superset-frontend/package.json

```diff
@@ -111,6 +111,7 @@
     "interweave": "^11.2.0",
     "jquery": "^3.4.1",
     "json-bigint": "^0.3.0",
+    "json-stringify-pretty-compact": "^2.0.0",
```

Comment:
There's no change to `package-lock.json` because this package is already included by `vega-lite` (run `npm ls json-stringify-pretty-compact`):

```text
├─┬ @superset-ui/preset-chart-xy@0.11.16
│ └─┬ vega-lite@4.1.1
│   └── json-stringify-pretty-compact@2.0.0  deduped
└── json-stringify-pretty-compact@2.0.0 
```